test/metabase/upload_test.clj

```diff
@@ -1747,8 +1657,8 @@
                               (update!)))))
                     (io/delete-file file)))))))))))
 
-(deftest update-coercing-multiple-columns-test
-  (mt/test-drivers (mt/normal-drivers-with-feature :uploads)
+(deftest update-promotion-multiple-columns-test
+  (mt/test-drivers (disj (mt/normal-drivers-with-feature :uploads) :redshift) ; redshift doesn't support promotion
```

Comment:
It would be nice to have a test covering *how* it fails, which will also prompt us to update the tests if we fix it.

Reply:
> how it fails 

Can you be more specific? Do you mean test the `driver/alter-columns!` method? I don't know what benefit that has